integrate-kobiton-travisci.md

```diff
@@ -0,0 +1,182 @@
+# Run Kobiton Automation Test on Travis CI
+
+## Table of contents 
+  - [Prerequisites](#prerequisites)
+  - [1. Configure Travis CI with GitHub repository](#1-configure-travis-ci-with-github-repository)
+  - [2. Configure automation test script](#2-configure-automation-test-script)
+  - [3. Run automation script on Kobiton devices](#3-run-automation-script-on-kobiton-devices)
+  - [4. Fetch session data through REST API](#4-fetch-session-data-through-REST-API)
+  - [5. (Optional) Travis CI customization](#5-optional-travis-ci-customization)
+  - [6. Feedback](#6-feedback)
+
+## Prerequisites
+  - Kobiton account
+  
+  Please visit https://portal.kobiton.com/register to create new account.
+
+## 1. Configure Travis CI with GitHub repository
```

Comment:
1. Need some introductions about Travis CI:
- What is Travis CI?
- Why do we need Travis CI?
- Application of Travis CI.
- ...

2. Regarding to the connection between Github / Travis CI / Integrating, As reader, I feel very hard to understand the flow & connection in between:
- E.x: How can only create file .travis.yml integrate with Travis CI?
--> Require more explanation for the connection (provide flow diagram if necessary)